apps/dashboard/app/(app)/logs/components/controls/components/logs-filters/components/paths-filter.tsx

```
@@ -134,6 +63,21 @@
     updateFilters([...otherFilters, ...pathFilters]);
   }, [checkboxes, filters, updateFilters]);
 
+  if (isLoading) {
+    return (
+      <div
+        className="flex flex-col items-center justify-center p-4"
+        role="status"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<output>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)